ckanext/harvest/harvesters/ckanharvester.py

```diff
@@ -48,6 +52,15 @@ def _get_group(self, base_url, group_name):
         except Exception, e:
             raise e
 
+    def _get_organization(self, base_url, org_name):
+        url = base_url + self._get_action_api_offset() + '/organization_show?id=' + org_name
+        try:
+            content = self._get_content(url)
+            content_dict = json.loads(content)
+            return content_dict['result']
+        except Exception, e:
```

Comment:
I hadn't noticed the line on 353. I still think that catch all exceptions are bad (even if wrote the `_get_group` one a long time ago :speak_no_evil: ) but I agree that we can let this one go in.